jxboot-resources/templates/700-chartmuseum-ing.yaml

```diff
@@ -11,14 +11,14 @@ metadata:
 {{- if .Values.chartmuseum.ingress.annotations }}
 {{ toYaml .Values.chartmuseum.ingress.annotations | indent 4 }}
 {{- end }}
-  name: chartmuseum
+  name: {{.Release.Name}}-chartmuseum
 spec:
   rules:
   - host: chartmuseum{{ .Values.cluster.namespaceSubDomain }}{{ .Values.cluster.domain }}
     http:
       paths:
       - backend:
-          serviceName: jenkins-x-chartmuseum
+          serviceName: {{.Release.Name}}-chartmuseum
```

Comment:
Not sure whether `{{.Release.Name}}` resolves to the value here https://github.com/jenkins-x/jenkins-x-platform/blob/90d20e869ef18843f5db375f2971f199559066e4/jenkins-x-platform/templates/chartmuseum-link-service.yaml#L5

Reply:
https://github.com/jenkins-x/jenkins-x-boot-config/blob/master/jenkins-x.yml#L223-L235 -> It passes `jenkins-x`, so I used the {{.Release.Name}}. I saw this charts created during the `install-jenkins-x` step.
The chart museum charts actually use `{{.Release.Name}}`.

Reply:
perfect 👍